edx_proctoring/constants.py

```diff
@@ -55,4 +55,23 @@
     else getattr(settings, 'SOFTWARE_SECURE_SHUT_DOWN_GRACEPERIOD', 10)
 )
 
+DEFAULT_CLIENT_POLLING_INTERVAL = (
+    settings.PROCTORING_SETTINGS['DEFAULT_CLIENT_POLLING_INTERVAL'] if
+    'DEFAULT_CLIENT_POLLING_INTERVAL' in settings.PROCTORING_SETTINGS
+    else getattr(settings, 'DEFAULT_CLIENT_POLLING_INTERVAL', 30)
```

Comment:
Yes, the idea was that the numbers would get 'fixed' for this PR when it was complete. I wanted to make sure the code was correct first, which seems like a good idea in retrospect.